py3status/modules/mpd_status.py

```diff
@@ -91,24 +101,48 @@ class Py3status:
     # available configuration parameters
     cache_timeout = 2
     format = '{state} [[[{artist}] - {title}]|[{file}]]'
-    hide_when_paused = False
-    hide_when_stopped = True
+    hide_when_pause = False
+    hide_when_play = False
+    hide_when_stop = True
```

Comment:
I got rid of them in `sf-mpd_status` branch. We don't keep configurations for small things like icons anymore (imho) as it's too simple, small, and maybe misleading. Should be more larger... `safe_format` strings... for more customization.

No more `[stop]`, `[play]`, and `[pause]`. We could use 3 full strings instead w/  same placeholders for more customization. Idk. You look at my branch again.

`'[\?color=#0f0&show \[play\] ][\?color=#0c0&show [[[{artist}] - {title}]|[{file}]]]'`

Can be for `format_play` or `state_play`. If we want to hide them, we use `state_play = ""` instead... or if you want to get crazy with the future formatter... Put everything on one format...  and pass states into the state placeholders.

`format = '{state_play}|{state_pause}|{state_stop}|{state_not_connected}|{state_not_authenticated}'`
---------- OR ----------
`format = '{play}|{pause}|{stop}|{not_connected}|{not_authenticated}'`

Users could get rid of unwanted states using two different ways. We check for them in `post_config_hook`. Look for `state = ""` or `not py3.format_contain`. Good solution?